openy.install

```diff
@@ -51,14 +51,110 @@ function openy_update_dependencies() {
   $dependencies['openy_node_alert'] = [
     8009 => [
       'system' => 8501,
+      'openy_node_session' => 8010,
     ],
   ];
-
+  
   $dependencies['openy_upgrade_tool'] = [
     8003 => [
       'simple_sitemap' => 8217,
     ],
   ];
+
+
+  $dependencies['redirect'] = [
+    8105 => [
+      'system' => 8403,
+    ],
+  ];
+
+  $dependencies['simple_sitemap'] = [
+    8216 => [
+      'block_content' => 8600,
+    ],
+  ];
+
+  $dependencies['webform_update'] = [
+    8157 => [
+      'block_content' => 8600,
+    ],
+  ];
+
+  $dependencies['page_manager'] = [
+    8401 => [
+      'taxonomy' => 8601,
+    ],
+  ];
+
+  $dependencies['openy_prgf_event_listing'] = [
+    8001 => [
+      'openy_node_session' => 8009,
+    ],
+  ];
+
+  $dependencies['openy_prgf'] = [
+    8008 => [
+      'openy_node_alert' => 8010,
+      'openy_node_session' => 8009,
+    ],
+  ];
+
+  $dependencies['openy_loc_facility'] = [
+    8008 => [
+      'openy_node_alert' => 8010,
+      'openy_node_session' => 8009,
+    ],
+  ];
+
+  $dependencies['openy_loc_branch'] = [
+    8016 => [
+      'openy_node_alert' => 8010,
+      'openy_node_session' => 8009,
```

Comment:
Since `openy_node_session_8009` won't be invoked at all because of the wrong dependency, `openy_loc_branch_8016` won't be invoked too.

Reply:
fixed above

Reply:
Now it's just excessive. 
`openy_node_alert_update_8010` depends on `openy_node_alert_update_8009` just by order
`openy_node_alert_update_8009` depends on `openy_node_session_update_8010` (see this PR above)
So you can leave just `'openy_node_alert' => 8010,`

But why would `openy_loc_branch` depend on `openy_node_alert`?